package.json

```diff
@@ -21,8 +21,9 @@
     "build": "react-app-rewired build",
     "eject": "react-app-rewired eject",
     "lint:ts": "eslint --ext .js,.jsx,.ts,.tsx .",
+    "lint:prettier": "prettier --write .",
```

Comment:
This shouldn't use `--write` as that will automatically fix the errors. Purpose of this script is to check whether there are any errors

Reply:
We probably should use `--check`

Reply:
Did that in the hurry, just copied `lint-staged` - my mistake. Should be good now